src/main/java/com/google/sps/servlets/GenerateServlet.java

```diff
@@ -108,6 +126,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     return filteredDestinations;
   }
 
+  /* Returns the filteredDestinations Set with the correct number of Destinations. */
+  public Set<Destination> correctNumDests(Set<Destination> filteredDestinations, int numPlaces) {
+    if (filteredDestinations.size() == numPlaces) {
```

Comment:
Do you think this would be better as `<=` in order to handle situations where there are fewer destinations than the requested amount?

Reply:
I handle the case where there are fewer Destinations than the requested amount in the else statement in the doPost function, because if this is the case, I can just immediately return an error. So, in this function, I already know that there are enough filteredDestinations.

Reply:
👍